src/main/java/org/folio/rest/impl/LoginAPI.java

```diff
@@ -341,9 +571,9 @@ public void postAuthnLogin(String userAgent, String xForwardedFor,
             PostgresClient.getInstance(vertxContext.owner(), tenantId).get(
                 TABLE_NAME_CREDENTIALS, Credential.class, new Criterion(useridCrit),
                 true, getReply-> {
+                  // TODO And now we're actually getting a user's credentials.
```

Comment:
Why is this a TODO = planned but not yet implemented?

Reply:
I sometimes leave notes about how the code is working to myself with a // TODO flag so I can remember to remove them. Missed this one. Will remove it.